.github/workflows/build-test-ship.yaml

```diff
@@ -99,24 +140,30 @@ jobs:
       # On release, build service images and push to ECR.
       - name: Configure AWS credentials
         uses: aws-actions/configure-aws-credentials@v1
-        if: ${{ github.event.release }}
+        if: |
+          github.event_name == 'push' && 
+          steps.check-version-change.outputs.appchange == 'true'
```

Comment:
Is this change in logic to facilitate deployment to Stage on merge to `dev` for prereleases? It seems like it, but don't want to assume.

Reply:
Yes. There are a number of changes like this, because fundamentally the workflow is changing from:

- Merge PR to dev
- Tag release
- Ship images
- Update helm chart
- Merge updated helm charts
- Deploy

to
- Update helm chart
- Merge PR to dev
- Wait for images/charts to ship
- Deploy to stage
- Promote helm prerelease
- Wait for charts to ship to production
- Deploy to production